kernel/packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -128,7 +130,14 @@ export async function getLoadablePortableExperience(data: {
       baseUrl: baseUrl,
       baseUrlBundles: '',
       contents: mappings,
-      icon: sceneJsonData.display?.favicon
+      icon: sceneJsonData.display?.favicon,
+      land: {
+        baseUrl: baseUrl,
+        baseUrlBundles: '',
+        cid: sceneUrn,
+        sceneJsonData: sceneJsonData,
+        mappingsResponse: mappingsResponse
+      }
```

Comment:
why is this necessary?

Reply:
With my last commit it is not already needed 👍 

The problem was that the `getSceneData()` function (in `RestrictedActions.ts`) needed to use `this.parcelIdentity.land.sceneJsonData` to check some conditionals. Now, with the `isPOrtableExperience` flag we don't need to call `getSceneData()` when the parcel identity is a PE.